swan-cern/values.yaml

```diff
@@ -148,10 +148,12 @@ swan:
           client_id: # placeholder, check secrets
           client_secret: # placeholder, check secrets
           oauth_callback_url: # placeholder, check secrets
-
           # skip refreshing tokens if already refreshed in last 110 minutes
           # this assumes tokens provided by keycloak are valid for 120 minutes
           auth_refresh_age: 6600
+          # Force old behaviour. This call is expensive and users most probably
```

Comment:
What is "old" here? Specify JH versions please (the one when it was not done, and the one we are updating to where we saw this new behaviour). I would put this info in the commit message, not in the yaml.

Reply:
It was in both places. But removed it from the chart.